### PR TITLE
Feature/org unit writing

### PR DESCRIPTION
**Pull request by cmoesgaard**
Implemented the usual writing magic for org units.
Addresses aren't handled as of yet - awaiting final implementation of address writing logic.
Renamed some of the mapping fields to something slightly more specific

### CODE REVIEW

File: tests/util.py [hunk 1]
```diff
@@ -357,6 +359,33 @@ def _perform_request(self, path, **kwargs):
...
+        list ordering
+        """
+        obj = copy.deepcopy(obj)
+        for outer_value in filter(lambda x: type(x) is dict, obj.values()):
+            for inner_key in outer_value:
+                outer_value[inner_key] = sorted(outer_value[inner_key],
+                                                key=common.get_effect_from)
+        return obj
+
+    def assertEqualLoRa(self, expected, actual):
```
Comment (danchr) [inline tests/util.py]: “LoRA” isn't terribly helpful in this regard — perhaps `assertRegistrationsEqual` instead? (That appears to be what you're actually want to compare.) Also, a docstring would be nice.

File: tests/util.py [hunk 2]
```diff
@@ -357,6 +359,33 @@ def _perform_request(self, path, **kwargs):
 
         return self.client.open(path, **kwargs)
 
+    def sort_inner_lists(self, obj):
```
Comment (danchr) [inline tests/util.py]: This method doesn't actually refer to or use `self` anywhere — I'd say this should be a regular function instead. If you only use it within the other method, just having it as an internal helper function should suffice.

File: tests/util.py [hunk 3]
```diff
@@ -357,6 +359,33 @@ def _perform_request(self, path, **kwargs):
...
+
+        This is purely to help comparison tests, as we don't care about the
+        list ordering
+        """
+        obj = copy.deepcopy(obj)
+        for outer_value in filter(lambda x: type(x) is dict, obj.values()):
+            for inner_key in outer_value:
+                outer_value[inner_key] = sorted(outer_value[inner_key],
+                                                key=common.get_effect_from)
+        return obj
```
Comment (danchr) [inline tests/util.py]: Personally, I'd use a `dict`-comprehension here, but that's just me 🙂

File: mora/service/org.py [hunk 4]
```diff
@@ -17,11 +17,16 @@
...
 
 import flask
 import werkzeug
 
-from . import common, keys
+from mora import lora
+from . import mapping
+from .common import (create_organisationsenhed_payload, _create_virkning,
+                     inactivate_old_interval, set_object_value, update_payload,
+                     ensure_bounds)
```
Comment (danchr) [inline mora/service/org.py]: I don't know why, but this style of import always seemed a bit messy to me…

File: tests/util.py (hunk 1 above)
Comment (cmoesgaard) [reply tests/util.py]: Yeah, I'm terrible at naming things. I'll fix it.

File: mora/service/org.py (hunk 4 above)
Comment (cmoesgaard) [reply mora/service/org.py]: I agree, that's why I fixed it. :)

Comment (danchr) [reply mora/service/org.py]: Yeah, I noticed that later 🙂

**Pull request merged**